### DISCUSSION TITLE
missing dependency update

### DISCUSSION THREAD

**Discussion by stephaniewang526** [renovatebot/renovate#12997] [Request Help] [answered]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Describe the bug

renovate-bot did not update updates in both places of the file so I had to manually update one dependency myself: https://github.com/googleapis/java-bigquery/pull/1735/commits/4518155f0c83d4269483d43e4c1ee20bb836d61d

The bot proposed only one change: https://github.com/googleapis/java-bigquery/pull/1735/commits/6e74bfb4b1b357745ec0677b544879f128270bb3

This did not happen the last time (both changes were proposed by the bot): https://github.com/googleapis/java-bigquery/pull/1715/files 

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by rarkins** [comment]
I think next time raise a discussion here before you make changes at your side. I've attempted to reproduce the problem using the `pom.xml` and it works as expected: https://github.com/renovate-reproductions/java-bigquery/pull/1

What happened is:
1. At time of the PR being raised, only the first dependency update was detected
2. The commits added by Google bots/staff *disabled* further updates by Renovate (this is by design)

![image](https://user-images.githubusercontent.com/6311784/145099924-e64601f6-e02b-48eb-bde0-761ef11f4aea.png)

**Reply by stephaniewang526** [reply]
Hi, I'm glad to hear it's working now when you tried to repro it. But I still wonder what happened at the time of the update. The screenshot is missing my action to update the dependency renovate-bot missed (the one done on 12/7/2021). Attaching a full screenshot here:

![image](https://user-images.githubusercontent.com/7338432/146063811-9fcd929f-cebc-4207-9504-360b9d5adc38.png)

I just want to make sure that the other dependency won't be missed next time since it results in an uppserbounddependency error when it's missed.

**Reply by rarkins** [reply]
It's quite common that differences in publishing times combined with caching of packages (particularly Renovate's caching) mean that brand new packages may not be detected at the same instant as each other. You should add a delay (e.g. stabilityDays=1 should work, prCreation=not-pending may also be enough to delay it by a few hours) or change your workflows so that you do not commit on top of Renovate PRs immediately after they are created. The commit on top of Renovate PRs is what means they are then locked and can't be updated later when the other package(s) are detected. If you are OK with Renovate force pushing to its own branches (and your own bots can then make their changes again each time) then you could add their email addresses to gitIgnoredAuthors (look it up on https://docs.renovatebot.com/configuration-options/)

Please let me know if any of the above is not clear.

**Comment by stephaniewang526** [comment, marked as answer]
Thanks -- we are addressing the issue by just importing the BOM: https://github.com/googleapis/java-bigquery/pull/1751/files
